docs/components_page/components/navbar/__init__.py

```diff
@@ -0,0 +1,93 @@
+from pathlib import Path
+
+import dash_core_components as dcc
+import dash_html_components as html
+
+from ...api_doc import ApiDoc
+from ...helpers import (
+    ExampleContainer,
+    HighlightedSource,
+    load_source_with_environment,
+)
+from ...metadata import get_component_metadata
+from .simple import navbar as navbar_simple
+
+GITHUB_EXAMPLES = "https://github.com/ASIDataScience/dash-bootstrap-components/blob/master/examples/advanced-component-usage/Navbars.py"  # noqa
+
+HERE = Path(__file__).parent
+
+navbar_simple_source = (HERE / "simple.py").read_text()
+navbar_source = (HERE / "navbar.py").read_text()
+
+
+def get_content(app):
+    return [
+        html.H2("Navbar"),
+        html.P(
+            dcc.Markdown(
+                "There are two navbar components in "
+                "*dash-bootstrap-components*, `NavbarSimple` and `Navbar`. "
```

Comment:
`Navbar`. `NavbarSimple` renders in a somewhat confusing way:

<img width="849" alt="screenshot 2019-01-06 at 12 32 56" src="https://user-images.githubusercontent.com/1392879/50736018-4e2c8400-11af-11e9-8fa4-647295f969be.png">

Maybe: "The `NavbarSimple` component easier to use but less flexible"?

Reply:
Good call, I'll change this then merge.